x-pack/plugins/triggers_actions_ui/public/application/sections/rule_form/rule_form.tsx

```diff
@@ -660,14 +660,22 @@ export const RuleForm = ({
       : false;
 
     if (MULTI_CONSUMER_RULE_TYPE_IDS.includes(rule?.ruleTypeId ?? '')) {
+      // Use selectedConsumer when creating a new rule, existing rule consumer when editing
+      const ruleConsumer = initialSelectedConsumer ? selectedConsumer : rule.consumer;
       return (
-        (validConsumers || VALID_CONSUMERS).includes(
-          selectedConsumer as RuleCreationValidConsumer
-        ) && hasAlertHasData
+        (validConsumers || VALID_CONSUMERS).includes(ruleConsumer as RuleCreationValidConsumer) &&
```

Comment:
Just a thought: can `validConsumers` be an empty array? And in that case, is it ok that we don't fall back to `VALID_CONSUMERS`?